### PR TITLE
Axis pr modification

### PR DESCRIPTION
**Pull request by oddbookworm**
This PR is exactly the same as #53, but with the required change made. I have not yet corrected the use of "colour" and "color" until it's discussed how we want to address that

### CODE REVIEW

File: intro/setup/index.html [hunk 1]
```diff
@@ -316,17 +316,54 @@ <h3>Exiting the game</h3>
 {% endhighlight %}
         
         And there we have it! Our game should close when we press the X button accordingly. 
+    </p>
+        
+    <p>
+        There are many different styles and ways to close a pygame application, 
+        you can raise an exception like shown here or use a variable to determine if the game is still running, 
+{% highlight python %}
+is_running = True
```
Comment (Matiiss) [inline intro/setup/index.html:325]: See below for a comment on this, TLDR: maybe name it `running` for simplicity while keeping readability

**Review by Matiiss — APPROVED** [review]
LGTM

**Comment by SSS-Says-Snek** [comment]
Yeah, I use `running` too

**Comment by SSS-Says-Snek** [comment]
LGTM too!

**Pull request merged**